pdc_client/runner.py

```diff
@@ -151,12 +152,15 @@ def setup(self):
 
     def run(self, args=None):
         self.args = self.parser.parse_args(args=args)
-        ca_cert_or_insecure = self.args.ca_cert or self.args.insecure
         if self.args.insecure:
             requests.packages.urllib3.disable_warnings(
                 requests.packages.urllib3.exceptions.InsecureRequestWarning)
-        self.client = pdc_client.PDCClient(self.args.server, page_size=self.args.page_size,
-                                           ca_cert_or_insecure=ca_cert_or_insecure)
+            ssl_verify = False
+        elif self.args.ca_cert:
+            ssl_verify = self.args.ca_cert
+        else:
+            ssl_verify = True
```

Comment:
Does it mean if not set related arguments, ssl_verify default is True? If so, in above class PDCClient part,
```
if ssl_verify is None:
     cfg_ssl_verify = config.get(CONFIG_SSL_VERIFY_KEY_NAME)
```
looks will not use argument in configuration file in this condition. Is it expected?